account_invoice_line_salesperson/models/account_invoice_line.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = "account.invoice.line"
+
+    salesman_id = fields.Many2one(
+        "res.users", "Sales Person", compute="_compute_salesman_id", store=True
```

Comment:
```suggestion
        "res.users", "Sales Person", compute="_compute_salesman_id", readonly=True, store=True
```

Revive `readonly=True` to be safe.